msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1287,9 +1298,9 @@ The bot receives an `installationUpdate` event when you install a bot to a conve
 > [!NOTE]
 > When you upgrade an application, and then add or remove a bot, the action also triggers the `installationUpdate` event. The **action** field is set to *add-upgrade* if you add a bot or *remove-upgrade* if you remove a bot.
 
-### Install update event
+When a new member is added to a channel, the `installationUpdate` event send an introductory message from your bot on installation. The bot also posts the message in the same channel using [conversationUpdate event](#team-members-added). This event helps you to meet your privacy and retain data requirements. You can delete user or the thread data when the bot is uninstalled.
```

Comment:
> When a new member is added to a channel, the `installationUpdate` event send an introductory message from your bot on installation.

This isn't right.

Reply:
Updated.

Reply:
I meant that the wording in the doc implies that the installationUpdate events **itself** sends an introductory message. 
> the `installationUpdate` event send an introductory message from your bot on installation

No: the installationUpdate event is simply an activity sent to the bot. The bot can use this event to send an intro message.